castai/provider.go

```diff
@@ -38,6 +38,7 @@ func Provider(version string) *schema.Provider {
 			"castai_eks_cluster":                resourceEKSCluster(),
 			"castai_eks_clusterid":              resourceEKSClusterID(),
 			"castai_gke_cluster":                resourceGKECluster(),
+			"castai_gke_cluster_id":             resourceGKEClusterId(),
```

Comment:
```suggestion
			"castai_gke_clusterid":             resourceGKEClusterId(),
```
Just to match naming style for `castai_eks_clusterid`

Reply:
I actually don't understand, how is it different from `castai_gke_cluster`?

Reply:
I don't think this is crucial, also it will cause changes in tf code as well.